webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/DirectiveApi.scala

```diff
@@ -551,8 +551,8 @@ class DirectiveApiService14 (
   def directiveTree(includeSystem : Boolean) : IOResult[JRDirectiveTreeCategory] = {
     def filterSystem(cat : FullActiveTechniqueCategory) : FullActiveTechniqueCategory = {
       cat.copy(
-          subCategories    = cat.subCategories.filter(c => includeSystem || c.isSystem).map(filterSystem)
-        , activeTechniques = cat.activeTechniques.filterNot(c => includeSystem || c.isSystem).map(t => t.copy(directives = t.directives.filterNot(_.isSystem)))
+          subCategories    = cat.subCategories.filter(c => includeSystem || !c.isSystem).map(filterSystem)
```

Comment:
It should be sorted too, to be consistent between run and help the user who is calling it to see where things are

Reply:
sorted by category and technique human readable name